bmi_calculator/bmi_calculator.py

```diff
@@ -0,0 +1,129 @@
+import sys
```

Comment:
Please provide the header format which includes ,   auther , reviewer , created  & modified date along with VDAC license  format